app/views/main.scala.html

```diff
@@ -59,6 +59,17 @@
         <div class="govuk-width-container ">
             <div class="govuk-footer__meta">
                 <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
+                    <ul class="govuk-footer__inline-list">
+                        <li class="govuk-footer__inline-list-item">
+                          <a class="govuk-footer__link" href="mailto:tdr@@nationalarchives.gov.uk">
```

Comment:
Is the double `@` sign a copy-paste error? Or is it some Play templating syntax?

Reply:
the double `@` is for mailto to work with the play framework https://stackoverflow.com/questions/42714889/mailto-link-not-working-using-play-framework-in-scala-html